src/adp/azext_adp/aaz/latest/adp/account/_create.py

```diff
@@ -0,0 +1,265 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "adp account create",
+)
+class Create(AAZCommand):
+    """Create an ADP account
+    """
```

Comment:
Please add examples for commands as well.

Reply:
normally for exported create and update cmds, examples are needed. Please add examples in your codegenv2 workspace and then export